.github/workflows/test.yml

```diff
@@ -39,7 +39,46 @@ jobs:
       - name: Test
         working-directory: build-${{ matrix.compiler }}-${{ matrix.build_type }}
         run: ctest --output-on-failure
-  
+
+  cpp-linux-arm64:
+    name: "C++ tests on Linux (arm64)"
+    runs-on:
+      - "self-hosted"
+      - "ARM64"
+      - "Linux"
+    strategy:
+      fail-fast: false
+      matrix:
+        build_type:
+          - Debug
+          - Release
```

Comment:
Very happy with the new runner! Good job.

As discussed, I suggest cutting this matrix down to
```
build_type:
  - Release
compiler:
  - g++
```

So that we have only one test build running on that poor raspberry pi.
Alternatively remove the matrix and replace `${{ matrix.compiler }}` with `g++` and `${{ matrix.build_type }}` with `Release`.

(I think "Release" mode is better for test coverage on this new architecture, to potentially catch issues that could arise specifically on ARM wrt e.g. compiler optimizations.)

Reply:
Done